src/components/InputAnt/InputAnt.jsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+import { Input } from "antd";
+import "./style.css";
```

Comment:
As you use styled-components approach for styling there is no need to use plan css